src/views/views.py

```diff
@@ -23,6 +25,33 @@
 ############################################################################################################
 # Account views
 ############################################################################################################
+from django.views import View
+from django.shortcuts import redirect, render
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+class add_link(LoginRequiredMixin, View):
+    def post(self, request):
+        link = request.POST.get('link')
+        if link:
+            UserLink.objects.create(user=request.user, link=link)
```

Comment:
It's forbidden to talk directly to data layer from UI layer, this needs to go tru Controller